apps/desktop-wallet/src/components/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { colord } from 'colord'
+import { AnimatePresence, motion } from 'framer-motion'
+import { Megaphone, X } from 'lucide-react'
+import { useState } from 'react'
+import styled from 'styled-components'
+
+import Button from '@/components/Button'
+import useThrottledGitHubApi, { storeAppMetadata } from '@/hooks/useThrottledGitHubApi'
+import { appHeaderHeightPx, messagesLeftMarginPx, walletSidebarWidthPx } from '@/style/globalStyles'
+import { Announcement } from '@/types/announcement'
+import { useTimeout } from '@/utils/hooks'
+import { links } from '@/utils/links'
+import { openInWebBrowser } from '@/utils/misc'
+
+import announcementFile from '../../announcement.json'
+
+interface AnnouncementBannerProps {
+  className?: string
+}
+
+const AnnouncementBanner = ({ className }: AnnouncementBannerProps) => {
+  const [announcement, setAnnouncement] = useState<Announcement | undefined>(
+    import.meta.env.VITE_USE_LOCAL_ANNOUNCEMENT_FILE === 'true' ? announcementFile : undefined
+  )
+
+  const [isCompact, setIsCompact] = useState(false)
+
+  useTimeout(() => setIsCompact(true), 5000)
```

Comment:
I had the experience where I moved my mouse to go click the button and the moment I was doing that the announcement got shrinked...

Then, when clicking the bubble, I couldn't see it anymore.

I liked the previous behavior more, where you could hide it by just clicking on it, otherwise it would remain visible.

<img width="692" alt="image" src="https://github.com/alephium/alephium-frontend/assets/1579899/06c1e00f-4a49-4f1a-b012-f394ea59b0f4">